internal/notifier/opsgenie.go

```diff
@@ -69,6 +70,7 @@ func (s *Opsgenie) Post(ctx context.Context, event eventv1.Event) error {
 		Message:     event.InvolvedObject.Kind + "/" + event.InvolvedObject.Name,
 		Description: event.Message,
 		Details:     event.Metadata,
+		Severity     event.Severity,
```

Comment:
```suggestion
		Severity:     event.Severity,
```

Reply:
I am not sure if this will work. For OpsGenie/JSM to use this field, it must go under the `Details` section (means `extraProperties` in Flux). Like `revision`. But I am not so familiar with golang, so don't know how to implement.

Reply:
Yeah i was wondering how to test this. I think it will just be added as a separate dynamic property, but i may be wrong.

Reply:
Just check the [JIRA API docs](https://developer.atlassian.com/cloud/jira/service-desk-ops/rest/v1/api-group-integration-events/#api-jsm-ops-integration-v2-alerts-post) for the available body fields. OG/Jira works with `priority`.

Reply:
> Yeah i was wondering how to test this. I think it will just be added as a separate dynamic property, but i may be wrong.

As the rest of the fields here in Flux matches the ones in the Jira API, I think this is a 1:1 relation.

And I think I missed the point that anyway the whole `event.Metadata` (means `spec.extraProperties` section) is inserted. So the main issue is IMHO: In this Flux opsgenie module section, it can only put it as separate parameter to the body. JSM can only access the keys under `Details` for usage in a rule. Maybe there is some other place where `event.Metadata` is created and there the severity can be added.